examples/simple_example/main.tf

```diff
@@ -21,6 +21,10 @@ provider "google" {
 module "folders" {
   source = "../.."
 
-  project_id  = "${var.project_id}"
-  bucket_name = "${var.bucket_name}"
-}
+  parent_id   = "${var.parent_id}"
+  parent_type = "${var.parent_type}"
```

Comment:
Can any of these values be hard-coded to simplify the demonstration of usage? For example, the example module could create a service account and assign that account as an admin.

Reply:
I have put all of them in variables so I can output them and declare them in the test inspec file. Is there any other way of reading this info in the test code?

Reply:
Terraform outputs are automatically provided as equivalently named InSpec attributes for tests, but there is no requirement that the values are provided as Terraform inputs.